docs: Update text in Module 17 Sharing code (#8657)
sveltejs/svelte: site/content/tutorial/17-module-context/01-sharing-code/text.md

Very occasionally, you'll need to run some code outside of an individual component instance. For example, you can play all five of these audio players simultaneously; it would be better if playing one stopped all the others.

> We can do that by declaring a `<script context="module">` block. Code contained inside it will run once, when the module first evaluates, rather than when a component is instantiated. Add this at the top of `AudioPlayer.svelte`, noting that we have to add a new script block as we only want the `current` variable as the only one affected by the context:

Reviewer: ```suggestion
We can do that by declaring a `<script context="module">` block. Code contained inside it will run once, when the module first evaluates, rather than when a component is instantiated. Add another script block to the top of `AudioPlayer.svelte`:
```

Author: Although shorter, I feel that an explanation here is needed for the student.
